README.md

```diff
@@ -1,10 +1,8 @@
 # matrix-appservice-mumble
 
-A simple Matrix to Mumble bridge. It sends messages between bridged rooms and tells you when people join / leave Murmur.
+A Matrix to Mumble bridge. It sends messages between bridged rooms and tells you when people join / leave Murmur.
 
-## Install
-
-### These instructions are for a development version, please use the instructions located [here](https://github.com/mymindstorm/matrix-appservice-mumble/tree/v0.2.0#install)
```

Comment:
Also, could you add this back and have it point to [here, where I have backported this](https://github.com/mymindstorm/matrix-appservice-mumble/tree/0.2.0-install-instructions#installation)?

Reply:
alrighty x)